util/fsdiffer/beforeafter.go

```diff
@@ -0,0 +1,63 @@
+package fsdiffer
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// The BeforeAfterFSDiffer is used to generate changes in a given directory
```

Comment:
s/The//
I don't particularly like the name BeforeAfterFSDiffer, but am struggling to think of an alternative right now..